tests/Feature/BulkDownloadTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Tests\Feature;
+
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\WithFaker;
+use Tests\TestCase;
+use KBox\Capability;
+
+
+class BulkDownloadTest extends TestCase
+{
+    /**
+     * A basic feature test example.
+     *
+     * @return void
+     */
+    public function test_bulk_download_when_no_data_sent()
+    {
+       $this->withoutExceptionHandling();
+       $user = tap(factory(\KBox\User::class)->create(), function ($u) {
+        $u->addCapabilities(Capability::$PARTNER);
+        });
+       
+       $response = $this
+            ->withHeaders([
+                'content-type' => 'application/json',
+            ])
+            ->actingAs($user)
+            ->json('POST', '/documents-download/');
+
+       $this->assertEquals(200, $response->status());
```

Comment:
Since you didn't specify any documents parameter here I would have expected a 422 status that indicate a [validation error](https://laravel.com/docs/7.x/http-tests#assert-session-has-errors).

```php
$response->assertSessionHasErrors('documents');
```

mainly because a zip cannot be created if you didn't select any documents